### ISSUE TITLE
package linux-zabbly missing on noble

### ISSUE THREAD

**Issue by heldchen** [zabbly/linux#20]
first of: thanks for your kernel releases, using it for many months now on my lenovo notebooks after struggeling with the mainline ubuntu kernel releases!

the noble releases seem to be currently missing the meta-package `linux-zabbly`, only the kernel `packages` are built:
```
$ apt-cache policy linux-zabbly
N: Unable to locate package linux-zabbly
```
```
$ apt-cache policy linux*zabbly*
linux-image-6.8.8-zabbly+:
  Installed: (none)
  Candidate: 6.8.8-amd64-202404301432-ubuntu24.04
  Version table:
     6.8.8-amd64-202404301432-ubuntu24.04 500
        500 https://pkgs.zabbly.com/kernel/stable noble/main amd64 Packages
linux-image-6.8.9-zabbly+:
  Installed: (none)
  Candidate: 6.8.9-amd64-202405031544-ubuntu24.04
  Version table:
     6.8.9-amd64-202405031544-ubuntu24.04 500
        500 https://pkgs.zabbly.com/kernel/stable noble/main amd64 Packages
linux-headers-6.8.9-zabbly+:
  Installed: (none)
  Candidate: 6.8.9-amd64-202405031544-ubuntu24.04
  Version table:
     6.8.9-amd64-202405031544-ubuntu24.04 500
        500 https://pkgs.zabbly.com/kernel/stable noble/main amd64 Packages
linux-headers-6.8.8-zabbly+:
  Installed: (none)
  Candidate: 6.8.8-amd64-202404301432-ubuntu24.04
  Version table:
     6.8.8-amd64-202404301432-ubuntu24.04 500
        500 https://pkgs.zabbly.com/kernel/stable noble/main amd64 Packages
```

**Comment by stgraber** [comment]
Oops, there was a tiny typo in the publishing script, should be fixed now!

**Comment by heldchen** [comment]
confirmed the package being available on 24.04 now. thanks alot for the quick fix & have a good weekend!
